app-arch/engrampa/engrampa-1.27.1.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+if [[ ${PV} == 9999* ]]; then
+	EGIT_REPO_URI="https://github.com/mate-desktop/${PN}.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/mate-desktop/${PN}/archive/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~loong ~riscv ~x86"
+fi
+
+inherit mate optfeature
+
+DESCRIPTION="Engrampa archive manager for MATE"
+HOMEPAGE="https://mate-desktop.org/ https://github.com/mate-desktop/engrampa"
+
+LICENSE="FDL-1.1+ GPL-2+ LGPL-2+"
+SLOT="0"
+IUSE="caja magic"
+
+DEPEND="
+	>=dev-libs/glib-2.50:2
+	>=dev-libs/json-glib-0.14
+	virtual/libintl
+	x11-libs/gdk-pixbuf:2
+	>=x11-libs/gtk+-3.22:3[X]
+	x11-libs/libSM
+	x11-libs/pango
+	caja? ( >=mate-base/caja-1.17.1 )
+	magic? ( sys-apps/file )
```

Comment:
From the diff:
```
-       dnl *******************************************
-       dnl This check can be removed with file >= 5.38
-       dnl *******************************************
-       AC_MSG_CHECKING([for zstd mime type])
-       AC_RUN_IFELSE([AC_LANG_PROGRAM([[#include <stdio.h>
-#include <string.h>
-#include <magic.h>]],
-               [[
-    FILE *fptr;
- ````

so I think we want to depend on >=5.38 for file now that they removed the check.